frontend/src/features/Train/features/Image/components/ImageTrainspace.tsx

```diff
@@ -10,6 +10,7 @@ import {
   StepButton,
```

Comment:
Can you add a screen recording of your feature in action for Image Trainspace and Tabular Trainspace? 

@prishavall

Reply:
[New Recording - 7/14/2023, 1:36:14 PM](https://vimeo.com/845362166/8e3e9edf59)

Reply:
can you add this link to the pr description

@prishavall